apollo-integration/src/test/kotlin/com/apollographql/apollo/IntegrationTest.kt

```diff
@@ -307,7 +307,7 @@ class IntegrationTest {
   @Throws(Exception::class)
   fun operationParseResponseWithExtensions() {
     val source = Buffer().readFrom(javaClass.getResourceAsStream("/HeroNameResponse.json"))
-    val (_, _, _, _, _, extensions) = HeroNameQuery().parse(source)
+    val (_, _, _, _, extensions) = HeroNameQuery().parse(source)
```

Comment:
Nit: I don't know why we were destructuring before but it loooks like doing `val extensions = HeroNameQuery().parse(source).extensions` is a more robust way to get the extensions?

Reply:
Yeah good point, for future refactors. Let me change that soon.